src/ONYXKEYS.ts

```diff
@@ -249,6 +249,9 @@ const ONYXKEYS = {
     /** Details on whether an account is locked or not */
     NVP_PRIVATE_LOCK_ACCOUNT_DETAILS: 'nvp_private_lockAccountDetails',
 
+    /** Last short lived token the user used to sign in */
+    NVP_SHORT_LIVED_TOKEN: 'nvp_short_lived_token',
```

Comment:
```suggestion
    LAST_SHORT_LIVED_TOKEN: 'lastShortLivedToken',
```

I think this name is more suited